extensions/pinboard/package.json

```diff
@@ -30,6 +37,14 @@
       "title": "API Token",
       "description": "Enter your API Token (Pinboard Settings > Password)",
       "placeholder": "Enter your API Token (Pinboard Settings > Password)"
+    },
+    {
+      "name": "constantTags",
+      "type": "textfield",
+      "required": false,
+      "title": "Tags for use when searching with constant tags",
```

Comment:
Can we shorten that title a bit? Just "Constant Tags" maybe and the description anyway tells the user more...

Reply:
Yep